ui/tests/integration/components/secret-engine/configuration-details-test.js

```diff
@@ -35,10 +37,8 @@ module('Integration | Component | SecretEngine/ConfigurationDetails', function (
       .hasText(`Get started by configuring your Display Name secrets engine.`);
   });
 
-  test('it shows config details if configModel(s) are passed in', async function (assert) {
-    assert.expect(36);
-    const allEnginesArray = allEngines(); // saving as const so we don't invoke the method multiple times via the for loop
-    for (const type of CONFIGURABLE_SECRET_ENGINES) {
+  for (const type of CONFIGURABLE_SECRET_ENGINES) {
+    test('it shows config details if configModel(s) are passed in', async function (assert) {
```

Comment:
to round out the debugging benefit i mentioned, we should interpolate the `type` in this test name 
```suggestion
    test(`it shows config details if configModel(s) are passed in for type: ${type} `, async function (assert) {
```